ansible_ai_connect/main/settings/base.py

```diff
@@ -539,6 +539,7 @@
 CSP_DEFAULT_SRC = ("'self'", "data:")
 CSP_INCLUDE_NONCE_IN = ["script-src-elem"]
 CSP_CONNECT_SRC = "'self'"
+CSP_STYLE_SRC = ("'self'", "'unsafe-inline'")
```

Comment:
Looks good to me as it seems to be a solution but for now I am not approving it till you undraft the issue.